### DISCUSSION TITLE
Support both local login/register as well as via discord

### DISCUSSION THREAD

**Discussion by sanegaming** [JakyeRU/Larascord#131] [Ideas]
My project requires both Local Auth as well as Discord OAuth as some of our users do not have discord.

I also am wanting to use Laravel with Jetstream/Livewire + Spatie Permissions, is breeze also required if I want to use those features? 
If I need to just use this project as inspiration and modify it to fit that works but just was curious on thoughts of this being a possibility.

**Comment by ValentinNeaga** [comment]
Hello,

I used Breeze just for the UI, it is not required. There are no plans for switching to Jetstream.
You can fork Larascord and modify it to suit your needs.

**Reply by sanegaming** [reply]
It seems required as any time I install it also pulls breeze. How can I install without the ui elements?
